src/modules/balance/handlers/handlerBalanceHistory.mixin.js

```diff
@@ -0,0 +1,76 @@
+import { getBalanceHistory } from '@/apollo/queries/wallets/wallets.graphql';
```

Comment:
We need to add  datapoints 0, to account for when balance was zero. So we still get actual line, rather then empty graph

Reply:
i added an update so it should work like what we talked about yesterday!